tests/x11/libreoffice/libreoffice_open_specified_file.pm

```diff
@@ -44,8 +45,8 @@ sub run {
             send_key "alt-o";
         }
         # close document
-        send_key "ctrl-f4";
-        wait_still_screen(2);
+        record_soft_failure("bsc#1209179 ctrl+f4 can't close libreoffice document") if is_tumbleweed;
```

Comment:
I think there is no need to record_soft_failure here.  This also happens on SLE15SP4 with new libreoffice updates.

Reply:
Or you can send_key first, then check_screen, if it fails, record_soft_failure then assert_and_click.

Reply:
I have added a needle as test purpose before, you can re-use it:
+        assert_and_click('libre-office-close-doc') if check_screen("libreoffice-test-$tag");
+        wait_still_screen(2);

Reply:
Thank you guys for your comments.
Codes and verification runs have been updated.